### ISSUE TITLE
Environment should become a skill modifier

### ISSUE THREAD

**Issue by mirasrael** [StrangeLoopGames/EcoSuggestions#481]
In real world you have an environmental factor for your house. So you probably will not like to live close to garbage or in area with bad ecology. 
Currently pollution affects your progression by destroying biomes when climate changes, but this process is really slow and it may not ever happen for the server to fail in result of ecological catastrophe even if they actively pollute the world.
Polluting will be more noticeable if it personally will affect every player in the game. So for every of your rooms it will apply negative skill modifier depending of current pollution of own and closest chunks. For zero pollution you will have modifier x1, but with pollution growing it will be decreased going to zero.
For character it should work as well, so if you are working or staying in polluted area then you will have the same modifier depending of current location (in real life you are feeling bad when you're in polluted area).
It will allow players to feel pollution more dynamic and realistic.

**Comment by elfl0rd** [comment]
But need protecting clothes! :)

**Comment by Deantwo** [comment]
My only issue with this idea would be range of effect. If your neighbor is polluting the whole area, affecting you and other. But the guy don't care because he already has all the skill he needs.

I guess we need more law options to be able to prevent that, such as taxing people for the about of pollution they make, or other punishments.

**Comment by mirasrael** [comment]
@Deantwo Yes, it exactly what should happen. It will force people to think about environment, make push on other players who ignores ecology with laws. It should bring more involvement for players in this process.

**Comment by Ruthgar** [comment]
It`s a feature request, not an issue...

**Comment by mirasrael** [comment]
@Ruthgar it is exactly why this was posted in EcoSuggestions rather than in EcoIssues ;)

**Comment by johnkslg** [comment]
Interesting suggestion, I would like to make sure it syncs up with the ability to get wrapped up in your world and not realize that pollution is affecting you directly.  It affects the group more than the individual, and tying it directly to the individual more will make it harder to 'go nuts with capitalism', and burn the world down to turn a profit (which is what I want to be a main threat of the game).  I will keep this in mind though.

**Comment by DragoonWise** [comment]
Air pollution could affect player by upping calories consumption.
